  * debian/patches/ralt_switch_multikey.diff: New patch to enable again
    lv3:ralt_switch_multikey which had been dropped from xkeyboard-config.
    Closes: #364435  Thanks Klaus Ethgen

Not submitted yet.

Index: xkb-data/rules/base.o_s.part
===================================================================
--- xkb-data.orig/rules/base.o_s.part
+++ xkb-data/rules/base.o_s.part
@@ -23,6 +23,7 @@
   grp:sclk_toggle	=	+group(sclk_toggle)
   lv3:switch		=	+level3(switch)
   lv3:ralt_switch	=	+level3(ralt_switch)
+  lv3:ralt_switch_multikey	=	+level3(ralt_switch_multikey)
   lv3:lalt_switch	=	+level3(lalt_switch)
   lv3:alt_switch	=	+level3(alt_switch)
   lv3:menu_switch	=	+level3(menu_switch)
Index: xkb-data/rules/base.xml.in
===================================================================
--- xkb-data.orig/rules/base.xml.in
+++ xkb-data/rules/base.xml.in
@@ -2841,6 +2841,12 @@
           <_description>Press Right Alt key to choose 3rd level.</_description>
         </configItem>
       </option>
+      <option>
+        <configItem>
+          <name>lv3:ralt_switch_multikey</name>
+          <_description>Press Right Alt key to choose 3rd level, Shift+Right Alt key is Multi_Key.</_description>
+        </configItem>
+      </option>
     </group>
     <group allowMultipleSelection="false">
       <!-- Tweaking the position of the "Ctrl" key -->
Index: xkb-data/symbols/level3
===================================================================
--- xkb-data.orig/symbols/level3
+++ xkb-data/symbols/level3
@@ -17,6 +17,14 @@
   modifier_map Mod5   { ISO_Level3_Shift };
 };
 
+partial modifier_keys
+xkb_symbols "ralt_switch_multikey" {
+  key <RALT> {
+    type[Group1]="TWO_LEVEL",
+    symbols[Group1] = [ ISO_Level3_Shift, Multi_key ]
+    };
+};
+
 // special case or right Alt switch - for use with grp:alts_toggle
 //
 partial hidden modifier_keys
